MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2022, [YOUR NAME]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
+
```

Comment:
Great job so far, I noticed that you are not set up your `MIT.md` file. 
Please it would be better if you set up the file correctly or if you don't need a copyright and license notices for this project you should remove it, this way it won't confuse other people who want to use your repo